lib/queue/consumer.ts

```diff
@@ -122,13 +122,18 @@ export class Consumer implements QueueConsumer {
 				pgPool: this.pool,
 				concurrency: defaultEnvironment.queue.concurrency,
 				pollInterval: 1000,
-				logger: new Logger((_scope) => {
-					return _.noop;
-				}),
+				logger: new Logger(graphileLoggerToJellyfishLogger(logContext)),
 				taskList: {
-					actionRequest: async (result) => {
+					actionRequest: async (graphilePayload, helpers) => {
 						// TS-TODO: Update graphile types to support Task list type parmaeterisation so we don't need to cast
-						const payload = result as ActionRequestContract;
+						const payload = graphilePayload as ActionRequestContract;
+						helpers.logger.debug('Consuming job', {
```

Comment:
Why are we using graphile logger instead of the jellyfish logger here?

Reply:
I followed the recommendation on https://github.com/graphile/worker#logger ( _It's recommended that your tasks always use the methods on helpers.logger for logging so that you can later route your messages to a different log store if you want to._ ) and created an adapter that converts those entries to the JF logger at: